src/foam/dao/ManyToManyRelationshipDAO.js

```diff
@@ -25,20 +25,30 @@ foam.CLASS({
   documentation: 'Adapts a DAO based on a *:* Relationship.',
 
   properties: [
-    'junctionProperty',
+    'junctionProperty', // TODO(markdittmer): Is this needed?
```

Comment:
Yes, because you don't know if this is going from target to source or from source to target.